### DISCUSSION TITLE
Cerastar ZWN24 working? With SAT?

### DISCUSSION THREAD

**Discussion by olivluca** [emsesp/EMS-ESP32#2314] [Q&A]
As per the title my boiler is a Junkers Cerastar ZWN24. It's not in the  [list of supported devices](https://docs.emsesp.org/Boilers/) but I see it is mentioned in a [fixed issue](https://github.com/emsesp/EMS-ESP32/issues/659) so I think it should work.
I have no ems thermostat so I'd want to use it with [SAT](https://github.com/emsesp/EMS-ESP32/issues/2103).

**Comment by proddy** [comment]
I think it should work, but can't be certain.
